sotodlib/tod_ops/azss.py

```diff
@@ -187,7 +237,7 @@ def get_azss(aman, signal=None, az=None, range=None, bins=100, flags=None,
     if merge_model:
         aman.wrap(azss_model_name, model_sig_tod, [(0, 'dets'), (1, 'samps')])
     if subtract_in_place:
-        aman.signal = np.subtract(signal, model_sig_tod, dtype='float32')
+        aman[signal_name] = np.subtract(signal, model_sig_tod, dtype='float32')
     return azss_stats, model_sig_tod
 
 def subtract_azss(aman, signal=None, azss_template=None,
```

Comment:
Can you add an `in_place` type of argument to subtract_azss here?

Reply:
I am confused. Do you want another argument than `subtract_in_place`?

Reply:
No, in the `subtract_azss` function we don't have an in_place option. We need that to work for the preprocess method to work.

Reply:
Ah, I got it.

Reply:
Ok, I have added that option.
https://github.com/simonsobs/sotodlib/blob/4d8eddf206cfcce61d47a0b6b3fb0502715645d1/sotodlib/tod_ops/azss.py#L247-L285

Changed subtract_hwpss to the same style as well.
https://github.com/simonsobs/sotodlib/blob/4d8eddf206cfcce61d47a0b6b3fb0502715645d1/sotodlib/hwp/hwp.py#L484-L525